tests/tests.rs

```diff
@@ -589,7 +590,7 @@ async fn test_memory_storage() {
 #[tokio::test]
 async fn test_disk_storage() {
     let _ = env_logger::try_init();
-    let _ = rustls::crypto::ring::default_provider().install_default();
+    let _ = ring::default_provider().install_default();
```

Comment:
I checked locally, and it seems that the default provider doesn't need to be installed anymore because it's now specified explicitly. Can you remove these calls?

Reply:
Right! I removed those calls